packaging/hudi-flink-bundle/pom.xml

```diff
@@ -189,6 +190,12 @@
       <artifactId>flink-connector-kafka_${scala.binary.version}</artifactId>
       <scope>compile</scope>
```

Comment:
@danny0405 should we change the version of  `flink-connector-kafka_${scala.binary.version}` to `${flink11.version}` as well ?